framework/src/modules/token/commands/transfer_cross_chain.ts

```diff
@@ -99,7 +99,7 @@ export class TransferCrossChainCommand extends BaseCommand {
 
 			balanceCheck.set(
 				messageFeeTokenID,
-				(balanceCheck.get(messageFeeTokenID) ?? BigInt(0)) + params.amount,
+				(balanceCheck.get(messageFeeTokenID) ?? BigInt(0)) + params.messageFee,
```

Comment:
@shuse2 Should this bug fix be also merged to release branch?

Reply:
yes, let's create another issue for the bug and create PR against release branch

Reply:
Ok I assume the merge issues are taken into account correctly.